packages/docs/src/page-configs/services/breakpoints/page-config.ts

```diff
@@ -0,0 +1,59 @@
+import { ApiDocsBlock } from '@/types/configTypes'
+import { PageGenerationHelper } from '@/helpers/DocsHelper'
+import { TableData, TableColumn } from '@/components/DocsTable/DocsTableTypes'
+import { helpersCodeSetup, helpersCodeTemplate, configCode, bodyClassCode } from '@/page-configs/services/breakpoints/code-examples'
+
+const columnsHelpersTypes: TableColumn[] = [
+  'name',
+  { title: 'type', type: 'code' },
+  { title: 'description', type: 'markdown' },
+]
+
+const tableDataHelpersTypes: TableData = [
+  ['current', 'string<"xs" | "sm" | "md" | "lg" | "xl">', 'breakpoints.helpers.list.current'],
+  ['thresholds', 'Record<"xs" | "sm" | "md" | "lg" | "xl", number>', 'breakpoints.helpers.list.thresholds'],
+  ['width', 'number', 'breakpoints.helpers.list.width'],
+  ['height', 'number', 'breakpoints.helpers.list.height'],
+  ['xs', 'boolean', 'breakpoints.helpers.list.exactThreshold'],
+  ['sm', 'boolean', 'breakpoints.helpers.list.exactThreshold'],
+  ['md', 'boolean', 'breakpoints.helpers.list.exactThreshold'],
+  ['lg', 'boolean', 'breakpoints.helpers.list.exactThreshold'],
+  ['xl', 'boolean', 'breakpoints.helpers.list.exactThreshold'],
+  ['smUp', 'boolean', 'breakpoints.helpers.list.upHelper'],
+  ['mdUp', 'boolean', 'breakpoints.helpers.list.upHelper'],
+  ['lgUp', 'boolean', 'breakpoints.helpers.list.upHelper'],
+  ['smDown', 'boolean', 'breakpoints.helpers.list.downHelper'],
+  ['mdDown', 'boolean', 'breakpoints.helpers.list.downHelper'],
+  ['lgDown', 'boolean', 'breakpoints.helpers.list.downHelper'],
+]
+
+const block = new PageGenerationHelper(__dirname)
+
+const config: ApiDocsBlock[] = [
+  block.title('breakpoints.title'),
+  block.paragraph('breakpoints.about'),
+
+  block.paragraph('breakpoints.sizes.intro'),
```

Comment:
Maybe we can add some kind of small example here? To show current breakpoint etc (not a huge prio, but feels like a nice convenience for user)